tests/validators/number.test.ts

```diff
@@ -1,5 +1,9 @@
 import { ConstraintError, s, ValidationError } from '../../src';
 
+const safeInteger = 42;
+// eslint-disable-next-line @typescript-eslint/no-loss-of-precision
+const unsafeInteger = 242043489611808769;
```

Comment:
Shouldn't this use MAX_SAFE_INTEGER or w/e

Reply:
No, because it's *un*safe